typescript/infra/src/govern/ProxiedRouterGovernor.ts

```diff
@@ -23,13 +27,20 @@ export class ProxiedRouterGovernor<
 
   protected async mapViolationsToCalls() {
     for (const violation of this.checker.violations) {
-      if (
-        violation.type ===
-        ConnectionClientViolationType.InterchainSecurityModule
-      ) {
-        this.handleIsmViolation(violation as ConnectionClientViolation);
-      } else {
-        throw new Error(`Unsupported violation type ${violation.type}`);
+      switch (violation.type) {
+        case ConnectionClientViolationType.InterchainSecurityModule:
+          // TODO remove upon transferring ownership of HelloWorld in multisig txs
+          console.log('Skipping ISM violation');
+          // this.handleIsmViolation(violation as ConnectionClientViolation);
```

Comment:
Are we fixing this before or after merging?

Reply:
Yeah will fix this before merging, now that I have multisig txs signed I can go ahead w this